client/syntaxes/bitbake.tmLanguage.json

```diff
@@ -49,46 +49,55 @@
         "keywords": {
             "patterns": [
                 {
-                    "match": "\\s*(include|from|import|require|inherit|addtask|deltask|after|before|export|echo|if|fi|else|return|unset|print|or|fakeroot|EXPORT_FUNCTIONS|INHERIT)(?=\\s+|\\s*:)",
+                    "match": "(\\binclude|require|inherit|addtask|deltask|after|before|export|echo|if|fi|unset|print|fakeroot|EXPORT_FUNCTIONS|INHERIT\\b)(?=\\s+)",
```

Comment:
I think you intend to put the "\\b" out of the parenthesis: `\\b(stuff|thing)\\b`. Currently they only apply to "include" and "INHERIT".

Reply:
Thanks, I changed them and added unit tests for verifying the exclusion of the unwanted scope. 
Actually, some scopes were correct with the previous version (by manual scope inspection and programmatic tests). 
Appreciated if you could have some suggestions on my unit tests.